SDPi_Supplement/asciidoc/volume2/dev-30/tf2-dev-30.adoc

```diff
@@ -9,8 +9,7 @@
 
 ==== Scope
 
-This transaction is used to retrieve the MDIB of a <<actor_somds_provider>> a <<actor_somds_consumer>> is interested in.
-
+include::tf2-dev-30-summary.adoc[]

```

Comment:
Here another line is needed as otherwise Asciidoc won't be able to render the heading correctly.

As a general rule of thumb: headings should always be enclosed in empty lines:

```
Foo

## Heading

Bar
```